### PR TITLE
Qmaps 811

### PR DESCRIPTION
**Pull request by xem**
show tooltips on PJ markers hover

### CODE REVIEW

File: src/scss/includes/popup.scss [hunk 1]
```diff
@@ -1,4 +1,8 @@
 .poi_popup__container {
+
+  position: absolute;
+  z-index: 6;
```
Comment (mlnor27) [inline src/scss/includes/popup.scss]: I think we have a dedicated file for `z-index`es, you could eventually put it there to be in adequation with the purpose of this file 👍

File: src/adapters/poi_popup.js [hunk 2]
```diff
@@ -42,39 +46,49 @@ PoiPopup.prototype.addListener = function(layer) {
...
   let poi = await ApiPoi.poiApiLoad(layerPoi.properties.global_id, {simple : true})
   if(poi) {
-    if(this.popupHandle) {
-      this.popupHandle.remove()
-      this.popupHandle = null
-    }
-    let {color} = IconManager.get(poi)
-    let category = poiSubClass(poi.subClassName)
-    let hours = poi.blocks.find(block =>
+    this.showPopup(poi, event);
```
Comment (mlnor27) [inline src/adapters/poi_popup.js]: Sorry, It looks like we do not use semicolons in this project. It is pretty unpleasing to make the devs go back to their PRs just for those kind of things. Maybe we could introduce a code formatter as a precommit hook in the future or eslint

File: src/adapters/poi_popup.js [hunk 3]
```diff
@@ -42,39 +46,49 @@ PoiPopup.prototype.addListener = function(layer) {
...
+  }
+}
+
+PoiPopup.prototype.createPJPopup =  function (poi, event) {
+  if(poi) {
+    this.showPopup(poi, event);
```
Comment (mlnor27) [inline src/adapters/poi_popup.js]: Again, for the semi-colon 👍

File: src/adapters/poi_popup.js (hunk 2 above)
Comment (xem) [reply src/adapters/poi_popup.js]: ;)

**Pull request merged**